manage.py

```diff
@@ -37,6 +39,21 @@ def add_event_identifier():
         save_to_db(event)
 
 
+@manager.command
+def fix_event_and_speaker_images():
+    events = Event.query.filter(Event.original_image_url.isnot(None),
+                                or_(Event.thumbnail_image_url == None, Event.large_image_url == None,
+                                    Event.icon_image_url == None)).all()
+    for event in events:
+        resize_event_images_task.delay(event.id, event.original_image_url)
+
+    speakers = Speaker.query.filter(Speaker.photo_url.isnot(None),
+                                    or_(Speaker.icon_image_url == None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4634410)